### DISCUSSION TITLE
Clawback pending

### DISCUSSION THREAD

**Discussion by Split4second** [Chia-Network/chia-blockchain#15699] [Support]
Hi,

I have clawback pending for 3 days.  Its not working. I full synced node and still cant get my chia back. 
Can someone help me.
I am new at this please so please write like writing to a child :)
Using Windows graphic GUI.
How to unstuck clawback function form pending
My version of chia is 1.8.2 latest 
Thanks
Vedran

**Comment by BrandtH22** [comment]
Hey Vedran,
I am sorry for the delayed response, currently the majority of our support is being provided through our discord server:
https://discord.gg/chia

As for the clawback a bit of additional context will help us to help you:
Are you getting any errors?
Is the option not available?
Did you send the clawback or did someone else send it to you?
Has the time expired on the clawback?

**Comment by Split4second** [comment]
Non of above just did not get it back to wallet.

**Reply by BrandtH22** [reply]
Can you provide more information by what you mean when saying it is pending?
Does it show as a pending clawback?
Also, did you send the clawback to someone or did someone send it to you?

Please do note that we are currently providing much faster response times through our discord server and we are happy to assist in getting this resolved:
https://discord.gg/chia
